scss/components/_buttons.scss

```diff
@@ -0,0 +1,144 @@
+//header_button
+
+header {
+  .submit_button {
+    height: 20%;
+    width: 15%;
+    border-radius: 10px;
+    margin: 2rem 1rem;
+    font-weight: 200;
+    line-height: 3rem;
+    font-size: 1rem;
+    border: $secondary-color-orange solid 1px;
+    text-transform: uppercase;
+    background-color: $secondary-color-orange;
+    color: white;
+    cursor: pointer;
+  }
+}
+
+//hero_button
+
+#hero {
```

Comment:
to samo....
tworz generyczne buttony ktore bedziesz uzywac na calej stronie a nie zamykasz je tylko do sekcji hero